src/app/components/navigation/navigation.component.html

```diff
@@ -1,13 +1,13 @@
 <div>
-    <mat-toolbar color="primary" id="pa_header">
-        <a *ngIf="!plugin.isPlugin" href="https://platform.planqk.de/my-content">
+    <mat-toolbar color="primary" [id]="qhanaService.isPlugin ? 'pa_header' : 'pa_header_pqk'">
```

Comment:
Ids should not be changed to achieve styling differences. It is always better to use css classes if styles should change!